backend/prisma/generated/models/Template.model.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
```

Comment:
I'll need explanation on models

Reply:
those models are generated automatically by the prisma models client, nothin to learn here tbh ;)